<launch>
  <!--include file="$(find dji_sdk)/launch/sdk.launch"/-->
  <include file="$(find hd_control)/launch/usb_cam.launch"/>
  <include file="$(find apriltags)/launch/apriltags.launch"/>
  <include file="$(find hd_control)/launch/pid.launch"/>
  <!--node name="hd_state_publisher" pkg="hd_control" type="drone_state_publisher_node" output="screen" /-->

  <node name="position_track_setpoint" pkg="hd_control" type="position_track_setpoint_node" output="screen" >
    <param name="tag_36h11_detection_topic" value="/apriltags/detections" />
    <param name="tag_16h5_detection_topic" value="/apriltags/16h5/detections" />
    <param name="landing_threshold_36h11" value="1.0" />
    <param name="landing_threshold_16h5" value="0.1" />
    <param name="reland_height_min_threshold" value="1.5" />
    <param name="reland_height_max_threshold" value="2.5" />
  </node>

  <node name="hd_drone_control" pkg="hd_control" type="drone_control_node"  output="screen"> 
      </node>

</launch>
